### DISCUSSION TITLE
How to monitor tensorboard logged scalar in modelcheckpoint?

### DISCUSSION THREAD

**Discussion by ghost** [Lightning-AI/pytorch-lightning#7184] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
If I save scalar log like this,
```
self.logger.experiment.add_scalars(‘loss/nll’, {‘train’: trainloss, ‘valid’: validloss})
```
How to monitor valid loss in Modelcheckpoint?

**Comment by justusschock** [comment, marked as answer]
you need to call it with self.log instead (we automate the rest for you), so that we are aware of you logging it :)

**Reply by ghost** [reply]
If one wants to group them and plot them into single figure to compare, one should log like i mentioned as https://github.com/PyTorchLightning/pytorch-lightning/issues/3932#issuecomment-704836037

So there is no way to monitor if one logs with `self.logger.experiment`?

**Reply by justusschock** [reply]
No, because we actually cache the values for monitoring and if you directly use `self.logger.experiment` you bypass our mechanism there which results in us not knowing about that value at all
